src/image.c

```diff
@@ -171,23 +171,30 @@ static void wolfBoot_verify_signature(uint8_t key_slot,
         struct wolfBoot_image *img, uint8_t *sig)
 {
     int ret, verify_res = 0;
-    uint8_t *pubkey = keystore_get_buffer(key_slot);
-    int pubkey_sz = keystore_get_size(key_slot);
-    int point_sz = pubkey_sz/2;
     ecc_key ecc;
-    mp_int r, s;
+    mp_int  r, s;
```

Comment:
```src/image.c:170:47: error: unused parameter 'key_slot' [-Werror,-Wunused-parameter]
  170 | static void wolfBoot_verify_signature(uint8_t key_slot,
      |                                ```

Reply:
```lib/wolfHSM/src/wh_client_cryptocb.c:497:30: error: unused parameter 'devId' [-Werror,-Wunused-parameter]
  497 | static int _handleSha256(int devId, wc_CryptoInfo* info, void* inCtx,
      |                              ^```

Reply:
fixed the warning in wolfBoot `src/image.c`, will fix the wolfHSM warning in a wolfHSM PR